learning/import/learning_importer.py

```diff
@@ -186,7 +186,7 @@ def convert_table(self, table_tag, text):
 
         self.markdown_to_html(table_tag, text)
 
-        return "\n\n" + table_tag.prettify() + "\n\n" + " ".join(footnotes) + "\n\n"
+        return "\n\n" + table_tag.prettify() + "\n\n- " + "\n- ".join(footnotes) + "\n\n"
```

Comment:
Hey Tim, I am not sure whats happening here with the python importer line. Also i thought these python importer files where only to migrate the initial website to markdown. Do we keep the importer files updated to automate the changes if we needed to import again?

Reply:
What this is doing is taking all the footnotes that will be added to the end of the table (e.g. `["[^1]","[^2]]`) and putting them all on separate lines that start with a `-` - this is what join does. E.g. 
```list_of_strings = ["hello", "goodbye", "so long"]
output = " ".join(list_of_strings)
print(output)```
would print `hello goodbye so long`. 
```list_of_strings = ["hello", "goodbye", "so long"]
output = " spacing ".join(list_of_strings)
print(output)```
would print `hello spacing goodbye spacing so long`.
```list_of_strings = ["[^1]", "[^2]", "[^3]"]
output = "\n- ".join(list_of_strings)
print(output)```
would output:
```- [^1]
- [^2]
- [^3]```

We could get away without making these updates to the importer but I just wanted to add these changes in to future proof it in case it's used to import any new modules in the future!